docs/SPEC.md

```diff
@@ -511,8 +475,6 @@ The string type name is `string`.
 The string type is nullable.
 Note that an empty string is distinct from a _null_ value.
 
-The length of a string is its size in bytes, not the number of characters, since a single character may be multiple bytes.
```

Comment:
Nothing in the SPEC depends on the length of a string as such this is best left documented in the stdlib strings package.